docs/configuration/environment.md

```diff
@@ -171,7 +171,10 @@ uv respects the following environment variables:
 - <a id="TRACING_DURATIONS_FILE"></a> [`TRACING_DURATIONS_FILE`](#TRACING_DURATIONS_FILE): Use to create the tracing durations file via the `tracing-durations-export` feature.
 - <a id="RUST_LOG"></a> [`RUST_LOG`](#RUST_LOG): If set, uv will use this value as the log level for its `--verbose` output. Accepts
   any filter compatible with the `tracing_subscriber` crate.
-  For example, `RUST_LOG=trace` will enable trace-level logging.
+  For example:
```

Comment:
This file is generated from `crates/uv-static/env_vars.rs`, you should change that file instead.

Reply:
oops! thanks!

Reply:
Done, sadly the doc generator removes the empty line after the bullet points, which will break the formatting :|